spec/models/order_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  before(:all) { User.create(fullname: "David Daniel", 
+                             email: "example@example.org",
+                             role: 0,
+                             phone: "222-333-4444",
+                             password: "password") }
+                           
+  let(:order) { Order.create(user_id: User.first.id, 
+                             status: "ordered",
+                              total: 1000) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 12, 44 should be on its own line.